### PR TITLE
test(datepicker): improve the coverage in datepicker component

### PR DESCRIPTION
**Pull request by YisusW**
- [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated tests.
 - [x] added/updated API documentation.
 - [x] added/updated demos.

### CODE REVIEW

**Comment by daniloff200** [comment]
Hi, @YisusW !
Thanks for your contribution! 
I do see a bunch of commits, which actually not related to this PR.
Could you squash them? 
I also see, that unit tests are failed, could you try to fix them, please?

**Comment by YisusW** [comment]
Hey @daniloff200,

i do not why, the ci report many errors, but i can not replicate in the local environment. i would like to know how to do it.

because i am doing changes to CI and hope it works because jest works okay in local.

**Comment by daniloff200** [comment]
@YisusW  I can't really say, what's going on right now, but, I've just used `npm run test` on my local and got the same errors, as on CI

**Comment by YisusW** [comment]
> @YisusW I can't really say, what's going on right now, but, I've just used `npm run test` on my local and got the same errors, as on CI

after merge the origin repo i see the errors i will fix it soon as possible

File: src/datepicker/datepicker.component.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export const DATEPICKER_CONTROL_VALUE_ACCESSOR: Provider = {
                       [customClass]="customClass"
                       [dateDisabled]="dateDisabled"
                       [dayDisabled]="dayDisabled"
-                      [onlyCurrentMonth]="onlyCurrentMonth"
+                      (onlyCurrentMonth)="onlyCurrentMonth"
```
Comment (SvetlanaMuravlova) [inline src/datepicker/datepicker.component.ts:44]: @YisusW i think it isn't really correct improve

**Pull request closed without merging**